src/editors/EditorPage.jsx

```diff
@@ -41,20 +40,17 @@ export default function EditorPage({
       blockId={blockId}
       studioEndpointUrl={studioEndpointUrl}
     >
-      <ModalDialog
-        title={blockType}
-        isOpen
-        size="fullscreen"
-        onClose={() => {}}
-        hasCloseButton={false}
-        variant="dark"
-      >
-        <div className="d-flex flex-column vh-100">
+      <div className="d-flex flex-column vh-100">
```

Comment:
You might be wondering-- why are we pretending this div is a modal? Well, for the purposes of getting these features to product, we need to flatten the z-index, as there is an ongoing issue with paragon's z-indexing of modals being too high. Consequently, at the future date where this needs to be a REAL modal again, that won't be a problem because paragon will have resolved it.<>

Reply:
Also changing to light theme for ease with title editing

Reply:
Yes, and until we get stacking modals, (which is coming), no dialogs or dropdown menus will work correctly inside modals if they are not in the react tree. Tinymce is one example of this.